unit-tests/live/d400/test-hdr.py

```diff
@@ -0,0 +1,638 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2024 Intel Corporation. All Rights Reserved.
+
+######################################
+# This set of tests is valid for any device that supports the HDR feature #
+######################################
+
+# test:device D400* !D457
+
+import pyrealsense2 as rs
+from rspy import test, log
+import time
+
+# HDR CONFIGURATION TESTS
+with test.closure("HDR Config - default config", "[hdr][live]"):
+    ctx = rs.context()
+    device = test.find_first_device_or_exit()
+    if ctx is not None:
+        devices_list = ctx.query_devices()
+        device_count = devices_list.size()
+        if device_count != 0:
+            depth_sensor = device.first_depth_sensor()
```

Comment:
This line can come right after
`device = test.find_first_device_or_exit()`
no need for lines
16,18-21
right?

Reply:
For all tests